src/Protobuf.elm

```diff
@@ -175,11 +180,36 @@ timestampEncoder v =
     JE.string <| ISO8601.toString <| ISO8601.fromTime <| round <| Time.inMilliseconds <| Date.toTime <| v
 
 
+{-| Turns a Result in to a Decoder
+Taken from <https://github.com/elm-community/json-extra/blob/2.7.0/src/Json/Decode/Extra.elm#L388>
+-}
+fromResult : Result String a -> JD.Decoder a
+fromResult result =
+    case result of
+        Ok successValue ->
+            JD.succeed successValue
+
+        Err errorMessage ->
+            JD.fail errorMessage
+
+
+{-| Decodes an Int from either a string or numeric.
+-}
+intDecoder : JD.Decoder Int
+intDecoder =
+    JD.oneOf [ JD.int, JD.string |> JD.andThen (String.toInt >> fromResult) ]
+
+
+numericStringEncoder : Int -> JE.Value
+numericStringEncoder =
+    toString >> JE.string
+
+
 {-| Decodes an IntValue.
 -}
 intValueDecoder : JD.Decoder Int
 intValueDecoder =
-    JD.int
+    intDecoder
 
 
 {-| Encodes an IntValue.
```

Comment:
IIRC this is currently used for both 32 and 64 bit wrapper types, ideally we should do the same thing as for the basic types, encoding 32 bit as numbers and 64 bit as strings. WDYT?

Reply:
Ah just noticed your comment in the conversation above mentioning this :) I'll go ahead and merge your current PR, let me know if you can also look into that, if not I will do it at some point. Thanks a lot!

Reply:
I didn't test this as extensively, but I think this functionality is already there. I was worried how clear it was, and I guess it isn't that clear.

I believe these changes in the go code will mean that wrapped 64 bit Ints will be emitted as strings: https://github.com/tiziano88/elm-protobuf/pull/12/files#diff-346e08d50aa913dadc9d982df75f248aL56

But, it is not symmetric between the 64 and 32 bit wrapped types. The 64 bit types have their encoders generated directly to `numericStringEncoder` but the 32 bit types go indirectly through intValueEncoder. I could tidy that up. Or, maybe I've missed something and the 64 bit wrapped types are broken at the moment?